nodebuilder/settings.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	fraudPkg "github.com/celestiaorg/celestia-node/fraud"
 	headerPkg "github.com/celestiaorg/celestia-node/header"
+	header "github.com/celestiaorg/celestia-node/nodebuilder/header"
```

Comment:
modhead. We aim to call all the modules with mod prefixed

Reply:
+1